torchvision/prototype/features/_bounding_box.py

```diff
@@ -15,7 +15,7 @@ class BoundingBoxFormat(StrEnum):
 
 
 class BoundingBox(Feature):
-    formats = BoundingBoxFormat
+    formats = BoundingBoxFormat  # Couldn't find a use of this in code. Is there a reason why we don't just let people access the enums directly?
```

Comment:
My thought was just convenience. I thought it would be more convenient to have something like

```py
from torchvision.prototype.features import BoundingBox

box = BoundingBox(..., format=BoundingBox.formats.XYXY)
```

rather than

```py
from torchvision.prototype.features import BoundingBox, BoundingBoxFormat

box = BoundingBox(..., format=BoundingBoxFormat.XYXY)
```

But since I never even used it myself and we also accept strings, e.g. `format="xyxy"`, we can probably remove it.